R/as_html.R

```diff
@@ -26,12 +26,15 @@ div_helper <- function(lst, class) {
 #' @param x `rtable` object
 #' @param class_table class for `table` tag
 #' @param class_tr class for `tr` tag
-#' @param class_td class for `td` tag
 #' @param class_th class for `th` tag
 #' @param width width
-#' @param link_label link anchor label (not including \code{tab:} prefix) for the table.
+#' @param link_label link anchor label (not including `tab:` prefix) for the table.
```

Comment:
I think we need a better explanation here

Reply:
(also for me ^^)

Reply:
This is so that you can link directly to the table if you add it to a website (e.g. `xyz.com/home#link-to-table`). Again, not sure if it's ever used but fine to leave in.